src/source-pcap-file-directory-helper.c

```diff
@@ -26,11 +26,13 @@
 #include "source-pcap-file-directory-helper.h"
 #include "runmode-unix-socket.h"
 #include "util-mem.h"
+#include "util-path.h"
 #include "source-pcap-file.h"
 
 static void GetTime(struct timespec *tm);
 static void CopyTime(struct timespec *from, struct timespec *to);
 static int CompareTimes(struct timespec *left, struct timespec *right);
+static TmEcode JoinPath (char *out_buf, uint16_t buf_len, const char *const dir, const char *const fname);
```

Comment:
@victorjulien This functionality doesn't exist elsewhere, correct? Mainly seeing snprintf elsewhere.

Reply:
Correct, we have no wrapper but we really need one. I would be in favor of adding it to `util-path.c` (as `PathJoin` instead of `JoinPath` :) ). Then we can open a ticket for applying it to all path handling.

Reply:
Sounds good to me.